src/Installer/Strings1.wxl

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WixLocalization Culture="en-us" xmlns="http://schemas.microsoft.com/wix/2006/localization">
+  <String Id="WelcomeDlgDescription">Welcome to the Npgsql for CrateDB setup wizard. This will install the Crate.io fork of Npgsql and the CrateDB-Plugin for Npgsql into your GAC and modify your machine.config.&#x0a;&#x0a;You should normally use Nuget to install Npgsql or the CrateDB-Plugin for Npgsql.</String>
```

Comment:
I would not mention that you are going to install the "fork" of Npgsql since this has no real value for a standard user who just wants to run PowerBI with CrateDB for example.

I suggest to write it as follows:

> This will install the Npgsql CrateDB Connector into your GAC.

Reply:
I agree. I changed the text accordingly.